### PR TITLE
[3.4] patch for dependency injection in controllers

### PR DESCRIPTION
**Pull request by DurandSacha**
Symfony packs a whole class, not objects !

### CODE REVIEW

File: controller.rst [hunk 1]
```diff
@@ -190,7 +190,7 @@ Templating and Twig are explained more in the
 Fetching Services
 ~~~~~~~~~~~~~~~~~
 
-Symfony comes *packed* with a lot of useful objects, called :doc:`services </service_container>`.
+Symfony comes *packed* with a lot of useful classes and functionality, called :doc:`services </service_container>`.
```
Comment (OskarStark) [inline controller.rst]: 👍  I like this rewording

File: controller.rst [hunk 2]
```diff
@@ -209,7 +209,7 @@ If you need a service in a controller, type-hint an argument with its class
         // ...
     }
 
-Awesome!
+Awesome! you can even get services from a construct function !
```
Comment (OskarStark) [inline controller.rst]: I am against merging this, it feels not helpful to me at this point

Comment (DurandSacha) [reply controller.rst]: @OskarStark Done :)

File: controller.rst (hunk 1 above)
Comment (HeahDude) [inline controller.rst]: functionalities?

Comment (DurandSacha) [reply controller.rst]: Done, Thanks !

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you Sacha.

**Pull request merged**